modules/Markbook/markbook_edit_addProcess.php

```diff
@@ -45,8 +45,8 @@
         $name = $_POST['name'] ?? '';
         $description = $_POST['description'] ?? '';
         $type = $_POST['type'] ?? '';
-        $date = (!empty($_POST['date']))? dateConvert($guid, $_POST['date']) : date('Y-m-d');
```

Comment:
Same situation as my comment about line 42 of `markbook_edit_addMultiProcess.php`.